plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -260,6 +381,72 @@ func BuildDimensions(mTags map[string]string) []*cloudwatch.Dimension {
 	return dimensions
 }
 
+func getStatisticType(name string) (sType statisticType, fieldName string) {
+	switch {
+	case strings.HasSuffix(name, "_max"):
+		sType = statisticTypeMax
+		fieldName = strings.TrimSuffix(name, "_max")
+	case strings.HasSuffix(name, "_min"):
+		sType = statisticTypeMin
+		fieldName = strings.TrimSuffix(name, "_min")
+	case strings.HasSuffix(name, "_sum"):
+		sType = statisticTypeSum
+		fieldName = strings.TrimSuffix(name, "_sum")
+	case strings.HasSuffix(name, "_count"):
+		sType = statisticTypeCount
+		fieldName = strings.TrimSuffix(name, "_count")
+	default:
+		sType = statisticTypeNone
+		fieldName = name
+	}
+	return
+}
+
+func convert(v interface{}) (value float64, ok bool) {
+
+	ok = true
+
+	switch t := v.(type) {
+	case int:
+		value = float64(t)
+	case int32:
+		value = float64(t)
+	case int64:
+		value = float64(t)
+	case uint64:
+		value = float64(t)
+	case float64:
+		value = t
+	case bool:
+		if t {
+			value = 1
```

Comment:
I see this is how it was before, but should this be `float64(1)` and `float64(0)` in order to preserve the type when sending to an output? cc @danielnelson

Reply:
This looks okay to me, value is a float64 and we are assigning a const to it.